### ISSUE TITLE
small issue in cocos2d-x/cocos/scripting/js-bindings/proj.android/Android.mk

### ISSUE THREAD

**Issue by scp-r** [cocos2d/cocos2d-x#16032]
in line 53 of cocos2d-x/cocos/scripting/js-bindings/proj.android/Android.mk:

```
                   ../auto/js_bindings_chipmunk_functions.cpp \
                   ../manual/ScriptingCore.cpp \ \
                   ../manual/cocos2d_specifics.cpp \
```

**Comment by minggo** [comment]
Thanks, i will remove the extra `\`.

**Comment by minggo** [comment]
fixed in #16035
